lib/src/features/helper/dec_string.dart

```diff
@@ -8,15 +8,15 @@ class DecString {
   static const MAX = 999999999999999999;
   static const ONE = 100000000000000000.0;
 
-  static String decStringFromDouble (double f) {
+  static String decStringFromDouble(double f) {
     return (f * ONE).toStringAsPrecision(PRECISION).split('.').first;
   }
 
-  static double doubleFromDecString (String f) {
+  static double doubleFromDecString(String f) {
     var i = int.parse(f);
```

Comment:
same here, counter, or anything that indicates what this variable will be used for is better than "i".

Reply:
@afti-githobo  written code.